### ISSUE TITLE
Inplace_abn not work for nshot=1

### ISSUE THREAD

**Issue by HieuPhan33** [fcdl94/FSS#6]
Hi,
when training for nshot=1,
I encounter the error `inplace_abn is trying to compute batch statistics, but the input tensor only contains a single sample per channel`.

The BatchNorm seems to raise error when batch size=1. Could you please suggest a solution for this problem?

Thanks.

**Comment by HieuPhan33** [comment]
No worries, I figure out the `no_pooling` option.

**Comment by fcdl94** [comment]
Yep, I should have commented it in the scripts :)
